docs/cloud-native-security/cloud-native-security-index.asciidoc

```diff
@@ -1,7 +1,38 @@
 [[cloud-native-security-overview]]
-
 = Cloud native security
-Elastic’s cloud security capabilities help you to improve your cloud security posture by comparing your cloud configuration to best practices, and allow you to monitor and investigate your Linux deployments inside and outside of Kubernetes.
```

Comment:
It's better to say ```Elastic Security for Cloud``` rather than ```Elastic’s cloud security``` for consistency across docs, blogs and other materials.